# we pass true so already defined exceptions are thrown
Run("calculator.cf")
Run("convertStr.cf")
Run("fizzBuzz.cf")
Run("randomNumberGuessr.cf")
Run("stairsMaker.cf")